user-guide/Cloud_Platform/Catalog/Tutorials/Tutorial_Register_Catalog_Item.md

```diff
@@ -0,0 +1,166 @@
+---
+uid: Tutorial_Register_Catalog_Item
+---
+
+# Registering a new catalog item
+
+This tutorial demonstrates how to add a new catalog item using the [Catalog API](xref:Catalog_Registration).  
+We will be registering our own version of the following example [connector](https://github.com/SkylineCommunications/SLC-C-Example_Rates-Custom).  
+
+For ease of use we will be using [Postman](https://www.postman.com/) to execute the register API calls. Feel free to integrate them afterwards in your workflow, e.g by using them in a github action.
+
+## Prerequisites
+
+- Organization key with permission "Register catalog items".
+
+## Overview
+
+- [Step1: Register catalog item URL](#step1-register-catalog-item-url)
+- [Step2: Authentication header](#step2-authentication-header)
+- [Step3: Register catalog item body](#step3-register-catalog-item-body)
+- [Step4: Register](#step-4-register)
+
+## Step1: Register catalog item URL
+Add a new request with http method **PUT** and following url <https://api.dataminer.services/api/key-catalog/v1-0/catalog/register>
+
+![Register item http url](~/user-guide/images/tutorial_catatalog_registration_item_url.png)
+
+## Step2: Authentication header
+The catalog item register api call uses Organization key authentication, we can obtain one in the [Admin App](https://admin.dataminer.services/) on the Keys page. 
+This key identifies your organization and will make sure the registration will register your catalog item under the organization of the key you use.
+
+> [!Important]  
+You need to have the "Owner" role in order to access/create Organization keys. 
+
+Go ahead and create a new key with permission "Register catalog items".
+
+![Organization Key](~/user-guide/images/tutorial_catatalog_registration_create_org_key.png)
+
+After creation of the key, you can copy the key and use it as value in the **Ocp-Apim-Subscription-Key** header  
+
+![Register item http header](~/user-guide/images/tutorial_catalog_registration_catalog_urlandheader.png)
+
+## Step3: Register catalog item body
+
+The body of the request expects form-data with key of type *File* and name *file*, the value is a zip containing
+
+* a readme.md file with any required image in an images folder which will be shown in the description tab in the Catalog.  
+Copy the below snippet and change the content as you see fit using your favorite editor.
+
+```md
+---
+uid: Connector_Example_Rates_Custom
+---
```

Comment:
uid is something we typically don't use in catalog itself.
This is only used (to my knowledge) on the docs site.

In the catalog app we even remove these lines when formatting